shop/src/app/globals.css

```diff
@@ -1,76 +1,78 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
- 
+
 @layer base {
   :root {
     --background: 0 0% 100%;
     --foreground: 222.2 84% 4.9%;
 
     --card: 0 0% 100%;
     --card-foreground: 222.2 84% 4.9%;
- 
+
     --popover: 0 0% 100%;
     --popover-foreground: 222.2 84% 4.9%;
- 
-    --primary: 222.2 47.4% 11.2%;
-    --primary-foreground: 210 40% 98%;
- 
+
+    /* GHO colors */
+    --primary: 258.62 47.54% 88.04%;
+    --primary-foreground: 245.29 16.5% 40.39%;
+
     --secondary: 210 40% 96.1%;
     --secondary-foreground: 222.2 47.4% 11.2%;
- 
+
     --muted: 210 40% 96.1%;
     --muted-foreground: 215.4 16.3% 46.9%;
- 
-    --accent: 210 40% 96.1%;
-    --accent-foreground: 222.2 47.4% 11.2%;
- 
+
+    /* GHO colors */
+    --accent: 258.62 47.54% 88.04%;
+    --accent-foreground: 245.29 16.5% 40.39%;
```

Comment:
In this file colors can be altered. Marking the ones I already changed with the comment `GHO colors`. In this case it's the accent and `primary` is the primary button style.